include/swift/AST/Attr.h

```diff
@@ -1858,6 +1858,89 @@ class DerivativeAttr final
   }
 };
 
+/// The `@transpose` attribute registers a function as a transpose of another
```

Comment:
nit: `@transpose(of:)`

Reply:
Done in https://github.com/apple/swift/pull/27545/commits/b94c097de935fd9a93b358aaf2d17f52c5e0b748.